.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -348,6 +349,8 @@ def instantiate_configs():
 
         # run docs builds on "pytorch-linux-xenial-py3.6-gcc5.4". Docs builds
         # should run on a CPU-only build that runs on all PRs.
+        # XXX should this be updated to a more modern build? Projects are
+        #     beginning to drop python3.6
```

Comment:
I think it's reasonable to move this to a more modern build. We could do that in a follow-up PR